lib/outbox/bandwidth/client.rb

```diff
@@ -38,7 +38,7 @@ def deliver(sms)
         messaging_client = @api_client.messaging_client.client
         body = ::Bandwidth::MessageRequest.new
         body.application_id = sms[:application_id]
-        body.to = sms.to
+        body.to = [sms.to]
```

Comment:
you should probably only wrap with an array if the `to` value is not already an array

Reply:
I'm pretty sure it will not be unless something changes in how we structure the messages. I tried sending setting it as an array in notifications and that did not work. I don't know if this is causing the problem or not, but I did see something about validations of the fields. I suspected there was something in between the notifications app and here that was causing a fuss so I changed it here. it wouldn't hurt to do it anyway in case that changes. I will also double check that sending an array from notifications does not work to make sure that was not a poorly executed test.

Reply:
yeap, that was a bad test. I will change this to support both arrays and strings. we can hold off on setting the recipient as an array in notifications for now.

Reply:
> we can hold off on setting the recipient as an array in notifications for now.

Yeah let's not change how we're structuring the recipient in the notifications app.

I just didn't want to get in a situation where we did want to send a group text for some particular reason and we had to come back and update this